README.md

```diff
@@ -17,94 +25,136 @@
+ - Create your first Dash app
+ - Live updates and debugging
 
-### Chapter 3: Dash Components and Layout
+### Chapter 3: Dash Components and Layouts
 
- - Core components: dcc.dropdown, dcc.graph 
- - HTML components: html.div
  - Design the app layout
-
-### Chapter 4: Callback I- connecting two components
-
- - Callback Decorator: Input, Output, Component_id, Component_property
- - Callback function
+ - Linking a CSS file
+ - Dash Core components
+   - Markdown
+   - Dropdown
+   - Checkbox
+   - Slider
+   - Graph
+   - etc...
+ - Dash Bootstrap Components:
+   - Container
+   - Accordion
+   - Card
+   - Layout
+   - Row and Col
+
+### Chapter 4: Linking Dash components
+
+ - Introduction to decorators in Python
+ - Callback decorators in Dash
+   - Input
+   - Output
+   - Component_id
+   - Component_property
+ - Setting up a callback
+   - Dropdown and Graph
+   - Slider and Graph
  
-### Chapter 5: Deploying app to the Web
+### Chapter 5: Deploying a Dash app to the Web
 
  - Basic deployment to Heroku
 
 ## Part II: Data Visualization
 
- - What you will learn in this part
+- Summary and overview of this part
+- Learning Intentions
 
-### Chapter 1: Using Data inside App
+### Chapter 6: Working with Data in Dash
 
- - Loading data: creating our own pandas df, from excel to df, connecting to api 
- 
-### Chapter 2: Wrangling data
+- Using an existing pandas dataframe
+- From excel or csv to pandas df
+- Creating our own pandas df 
+- Using data from an API
+
+### Chapter 7: Wrangling data
 
- - Intro to Pandas's basic operations
- - wrangling an example dataset 
+- Case Study: Select a dataset
+- Basic operations with pandas
+- Cleaning, processing, wrangling
+- Preprocessing files
 
-### Chapter 3: Data Visualization
+### Chapter 8: Data Visualization
 
- - Intro to Plotly Express (PX)
- - PX graphs: pie chart, bar chart, scatter plot (using fig.show())
- - Incorporting PX graphs in Dash app 
+ - Introduction to Plotly Express
+ - Plotly Express graphs
+   - scatter plot
+   - line plot
+   - bar plot
+   - etc...
+ - Principles of Effective Visualizations
+ - Incorporting PX graphs in a Dash app 
+ - References and resourcres
 
-## Part III: Dash
+## Part III: Polishing your Dash application
 
- - What you will learn in this part
+- Summary and overview of this part
+- Learning Intentions
  
-### Chapter 1: Advanced Layout and Styling
+### Chapter 9: Advanced Layout and Styling
 
  - Creating a layout component inside a callback
- - Dash Bootstrap Components: dbc.Container, dbc.Row, dbc.Col
- - Dash Bootstrap Components: Accordion, alert, input
+ - More Dash Bootstrap Components
+ - Dash HTML components
+ - CSS (Optional)
   
-### Chapter 2: Advanced Callback
+### Chapter 10: Advanced Callbacks
 
- - State
- - multiple buttons: callback_context
+ - States
+ - Multiple buttons: callback_context
  - Multiple outputs and inputs
 
-### Chapter 3: Advanced Components
+### Chapter 11: Advanced Components
 
  - RadioItem
```

Comment:
We can take out RadioItem from here. That was my mistake. That's not an advanced component.

Reply:
done